File: java/link-message-client/src/main/java/link/message/client/content/complex/linear/TitleLinearComplexMessageContentItem.java
package link.message.client.content.complex.linear;

import link.message.client.content.StringContent;
import link.message.client.utils.Guard;

import com.alibaba.fastjson.annotation.JSONField;

/**
 * 标题类型的线性消息内容项
 * 
 * @author zhongt
 *
 */
public class TitleLinearComplexMessageContentItem extends LinearComplexMessageContentItem {
	@JSONField(name="bgcolor")
	protected String  bgColor;
	@JSONField(name="font_color")
	protected String  fontColor;
	@JSONField(name="font_bold")
	protected boolean fontBold;
	@JSONField(name="border_color")
	protected String  borderColor;
	
	protected TitleLinearComplexMessageContentItem() {
		
	}
	
	public TitleLinearComplexMessageContentItem(String title) {
		Guard.guardReqiredString(title, "title must be set value.");
		
		this.content     = new StringContent(title);
		this.displayType = LinearComplexMessageContentType.TITLE;
	}

	public String getBgColor() {
		return bgColor;
	}

	public void setBgColor(String bgColor) {
		this.bgColor = bgColor;
	}

	public String getFontColor() {
		return fontColor;
	}

	public void setFontColor(String fontColor) {
		this.fontColor = fontColor;
	}

	public boolean isFontBold() {
		return fontBold;
	}

	public void setFontBold(boolean fontBold) {
		this.fontBold = fontBold;
	}

	public String getBorderColor() {
		return borderColor;
	}

	public void setBorderColor(String borderColor) {
		this.borderColor = borderColor;
	}

}

File: net/Link.Message.Client/content/complex/linear/TitleLinearComplexMessageContentItem.cs
using Link.Message.Client.utils;
using Newtonsoft.Json;

namespace Link.Message.Client.content.complex.linear
{
	/// <summary>
	/// 标题类型的线性消息内容项
	/// </summary>
	public class TitleLinearComplexMessageContentItem : LinearComplexMessageContentItem
	{
		protected internal string bgColor;
//JAVA TO C# CONVERTER TODO TASK: Most Java annotations will not have direct .NET equivalent attributes:
//ORIGINAL LINE: @JSONField(name="font_color") protected String fontColor;
		protected internal string fontColor;
//JAVA TO C# CONVERTER TODO TASK: Most Java annotations will not have direct .NET equivalent attributes:
//ORIGINAL LINE: @JSONField(name="font_bold") protected boolean fontBold;
		protected internal bool fontBold;
//JAVA TO C# CONVERTER TODO TASK: Most Java annotations will not have direct .NET equivalent attributes:
//ORIGINAL LINE: @JSONField(name="border_color") protected String borderColor;
		protected internal string borderColor;

		protected internal TitleLinearComplexMessageContentItem()
		{

		}

		public TitleLinearComplexMessageContentItem(string title)
		{
			Guard.GuardReqiredString(title, "title must be set value.");

			Content = new StringContent(title);
			DisplayType = LinearComplexMessageContentType.Title;
		}

        [JsonProperty("bgcolor")]
		public virtual string BgColor
		{
			get
			{
				return bgColor;
			}
			set
			{
				bgColor = value;
			}
		}

        [JsonProperty("font_color")]
		public virtual string FontColor
		{
			get
			{
				return fontColor;
			}
			set
			{
				fontColor = value;
			}
		}

        [JsonProperty("font_bold")]
		public virtual bool FontBold
		{
			get
			{
				return fontBold;
			}
			set
			{
				fontBold = value;
			}
		}

        [JsonProperty("border_color")]
		public virtual string BorderColor
		{
			get
			{
				return borderColor;
			}
			set
			{
				borderColor = value;
			}
		}


	}
}